src/utils.py

```diff
@@ -1264,12 +1294,17 @@ def create_pddl_problem(objects: Collection[Object],
 
 
 def create_video_from_partial_refinements(
-    task: Task, simulator: Callable[[State, Action], State],
-    render: Callable[[State, Task, Optional[Action]], List[Image]],
     partial_refinements: Sequence[Tuple[Sequence[_GroundNSRT],
-                                        Sequence[_Option]]]
+                                        Sequence[_Option]]],
+    env: BaseEnv,
+    train_or_test: str,
+    task_idx: int,
+    max_num_steps: int,
 ) -> Video:
-    """Create a video from a list of skeletons and partial refinements."""
+    """Create a video from a list of skeletons and partial refinements.
+
+    Note that the environment internal state is updated.
+    """
```

Comment:
it won't cause any problems that this method updates the internal state right? since it's only called from main.py which would always reset to a new initial state afterward?

Reply:
right